.github/workflows/ci-tests.yml

```diff
@@ -41,7 +41,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.7', '3.8', '3.9', '3.10', '3.11' ]
+        python-version: [ '3.7', '3.8', '3.9', '3.10' ]
```

Comment:
Would be nice to put some comment that old versions don't have coverage. So hopefully the next person, when 3.12 comes out, will know to put 3.11 in this list and put 3.12 on line 62.